specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/GetTransfer.json

```diff
@@ -1,6 +1,6 @@
 {
   "parameters": {
-    "api-version": "2018-08-01-preview",
+    "api-version": "2018-11-01-preview",
     "billingAccountName": "billingAccountName",
     "invoiceSectionName": "invoiceSectionName",
```

Comment:
Values for "recipientEmailId" and "initiatorEmailId" below should be different.

Reply:
Please fix in other transfer example files as well.

Reply:
Done.